bin/ch/ChakraRtInterface.h

```diff
@@ -309,7 +309,7 @@ class ChakraRTInterface
     static JsErrorCode WINAPI JsGetExternalData(JsValueRef object, void **data) { return HOOK_JS_API(GetExternalData(object, data)); }
     static JsErrorCode WINAPI JsCreateArray(unsigned int length, JsValueRef *result) { return HOOK_JS_API(CreateArray(length, result)); }
     static JsErrorCode WINAPI JsCreateArrayBuffer(unsigned int byteLength, JsValueRef *result) { return m_jsApiHooks.pfJsrtCreateArrayBuffer(byteLength, result); }
```

Comment:
Both were not in use for xplat static lib. This PR benefits `JsGetArrayBufferStorage` so it is updated.